android/src/main/java/com/rnfs/RNFSManager.java

```diff
@@ -734,11 +734,10 @@ public void onUploadBegin() {
       };
 
       params.onUploadProgress = new UploadParams.onUploadProgress() {
-        public void onUploadProgress(int fileCount,int totalBytesExpectedToSend,int totalBytesSent) {
```

Comment:
Is it a good idea to remove the FileID? What if i would to assign the upload progress event to a particular File upload running multiple of them?

Reply:
Also, the jobID-attribute is documented in README and part of the iOS-implementation also. Therefor we should keep it.

Reply:
Because IOS uploadFiles don't have FileID.
In this version, totalBytesSent and totalBytesExpectedToSend are depended on the file which is uploading now.
But IOS totalBytesSent and totalBytesExpectedToSend is depended on whole request length.
I think we should reach a consensus about the progressCallBack 😄

Reply:
Ah yes, probably a misconception between jobID and fileID.

on iOS (and in Readme), the jobID is mandatory:
https://github.com/itinance/react-native-fs/blob/master/RNFSManager.m#L601

Reply:
As long as we keep this also by for Android i am fine with the changes.

Reply:
Yeah!
I still keep jobID in progressCallBack.
[https://github.com/hank121314/react-native-fs/blob/master/android/src/main/java/com/rnfs/RNFSManager.java#740](url)